### ISSUE TITLE
how to impress the disk space utilization？

### ISSUE THREAD

**Issue by Coolboomer9** [JieSun-pku/detonationFoam#2]
The programming will be product a large of files after running modified 2D_detonation, and lead to disk space full.how to optimize?
Can I solve the problem through modifying the writeInterval term in controlDict?

**Comment by JieSun-pku** [comment]
Yes , just increase the writeInterval value in controlDict file.
